src/Charge.php

```diff
@@ -54,16 +54,17 @@ public function charge($id, array $form_params = [])
     /**
      * Make a POST request to charges/{id}/refund.
      *
-     * @param int $id The resource's id.
+     * @param int   $id             The resource's id.
+     * @param array $form_params    The request body.
      *
      * @return mixed
      * @throws \GuzzleHttp\Exception\GuzzleException
      * @throws \Vindi\Exceptions\RateLimitException
      * @throws \Vindi\Exceptions\RequestException
      */
-    public function refund($id)
+    public function refund($id,$form_params = [])
```

Comment:
Poderíamos padronizar com os demais métodos?
```suggestion
    public function refund($id, array $form_params = [])
```

Reply:
@rafaelyanagui consegue fazer essa alteração pra darmos seguimento no PR?